### DISCUSSION TITLE
My ts code is one in white

### DISCUSSION THREAD

**Discussion by zeMinng** [vuejs/language-tools#4914] [Q&A]
My ts code is one in white. 
I also installed Vue-Official
![image](https://github.com/user-attachments/assets/51af8ae8-5ae0-4b2f-a6d2-ca75fd981c7e)

**Comment by ilyaliao** [comment]
Can you take a screenshot of this?

<img width="347" alt="截圖 2024-10-09 18 34 31" src="https://github.com/user-attachments/assets/eb477da4-2e05-4c21-a579-237e228bb27f">

**Reply by zeMinng** [reply]
![image](https://github.com/user-attachments/assets/94d358fd-29e3-4ffa-9ac9-d413eb1517ec)

**Reply by zeMinng** [reply]
I checked out the reason, the problem in the settings.json. thanks
